src/Microsoft.ML.Core/Data/DenseVector.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.ML.Runtime.Data
```

Comment:
Will the type show in public surface area of mainline scenario APIs? If yes, should this type be in Microsoft.ML root?